proposals/all_proposals.md

```diff
@@ -0,0 +1,671 @@
+	- 17 liquid democracy
+- Data Security / Protection / Ownership
+	- 4, 8 regulate / ban surveillance companies
+	- *7, 13 European Autonomous Data Network
+	- *16 Public Digital Identity
+	- 12 require compensation for data generators
+	- 17 ban collection of personal data + proprietary software on personal communication devices
+	- 9 ratify Convention 108 of the Council of Europe + ePrivacy regulation 
+
+# 1 
+# 23.06.2018  Suggestion for a European Progressive Media Fund - 18-06-23 
+
+One of the suggestions I have on the media policy is the separation between content and platform. It is pretty straightforward (who write the content could not be the person which upload it on a platform) but it can be abstracted even more, to give a European perspective and ad to resist digital colonization.In short, we should produce content and it can go in a different digital platform. this will avoid to bound our community to a selected company. Imagine, if you have produced a text, your effort has been the content production. Then, is a matter of automation press "enter" and having it going to facebook, or to the Diem's blog, or to both of them? and why not elsewhere?The political proposition here is: "an entity, an institution, a political force, should not grow a community only in a convenient platform, but use all of them, automatically, as much as possible"In this way, we do not contribute in giving a unique value to the platform [ie: twitter has value because there are the tweets of donald trump], and we are not forcing our community to belong to a platform which capitalizes on users profiling.Now, let's do an effort in abstraction. Imagine the content production as a chain of tasks performed by different people, with different skills. Now I'll try to extrapolate the individual steps, they are rather hypothetical of course.A person start with a concept. a group writes a memo in flawless English after some legal/political revision. this group is producing an output (the text) and some additional metadata (stats reference, external evidence, legal analysis, links to the pictures which can be selected, contact of people, etc)the product can be translated into national languages, the translators should have access to the metadata, is for their own understanding of the context, or is additional value.then the content will look "technical" and someone else will do a copy-editing making a version more appealing for the audience.At this moment you have worked distributedly among nations, you need to have something which permits, in the least amount of formatting effort, to publish in all the platforms available.This last effort is more of an integration effort. If a technical development/research has to happen, is only in a tool which permits this content traceability, enhancing, fork-ing, translations, and the content production has to keep track of all the publications which has been doneon the platforms.
```

Comment:
This seems to be asking for an exchange standard, not a legislation. Standard document formats however exist, also means for upload — so if the industry hasn't established a standard practice in this field maybe that's because they have no economic incentive to do so? A legal policy could make it mandatory for publishing platforms to accept a uniform simple document upload format. Not clear to me which political problem this really addresses. - lynx